homeworks/05-redux-basic/redux-basic/src/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import { createStore, compose } from 'redux';
+import {balance} from '../src/redux/reducers/balanceReducer'
+import {types} from '../src/redux/actions/actionsTypes'
+
+
+declare global {
+  interface Window {
+    __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose;
+  }
+}
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
Store creation should be separated in different file and then imported in App to wrap React components with store provider.